New page set-up/Application settings (#3936)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Application-Settings/index.md

---

> versionFrom: 7.0.0

Author: Great :-)

---

# Custom Application Settings

Umbraco Cloud enables you to toggle a selection of predefined settings for each of the project's environments.

> A change of an application setting enables you to change one specific behavoir of the app service that an environment is running in. The objective with application settings are to empower you with feature or functionality that isn't available by default.

Reviewer: ```suggestion
A change of an application setting enables you to change one specific behavior of the app service that an environment is running in. The objective of application settings is to empower you with features or functionality that aren't available by default.
```

Author: Great suggestions. I accept! :-)

> Currently the only setting that currently is available is the _enable client certificate loaded fromm file system_.

Reviewer: ```suggestion
Currently, the only setting that is available is the _enable client certificate loaded from file system_.
```

Author: Great suggestions. I accept! :-)

## Enable client certificate loaded from file system Explained

> If your cloud project needs to load a client certificate (such as a X.509 certificate) at runtime you can turn on this feature for one or more environments.

Reviewer: ```suggestion
If your cloud project needs to load a client certificate (such as an X.509 certificate) at runtime you can turn on this feature for one or more environments.
```

Author: Is it really an X.509 and not a X.509?

Author: But is to, I accept :-)

> By turning this feature on for an environment you will be able to load a client certificate as a file during run-time of your cloud project.

Reviewer: ```suggestion
By turning this feature on for an environment, you will be able to load a client certificate as a file during the run-time of your cloud project.
```

Author: Great suggestions. I accept! :-)

![Enable Client Certificate](Images/EnableClientCertificateLoadedFromFileSystem.gif)

> You can find more information on loading a certificate from a file [here](https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-certificate-in-code#load-certificate-from-file).

Reviewer: ```suggestion
For more information on loading a certificate from a file, see the [Load certificate in Windows apps](https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-certificate-in-code#load-certificate-from-file) section in the Microsoft documentation.
```

Author: Great suggestions. I accept! :-)

:::note

> When toggling the setting you will be met by a confirmation prompt. An enabling/disabling of this feature will enforce a restart of the environment.

Reviewer: ```suggestion
When toggling the setting, a confirmation prompt is displayed. Enabling/disabling this feature enforces a restart of the environment.
```
